cloudformation/extraction.yaml

```diff
@@ -25,6 +26,14 @@ ExtractFunction:
               - sqs:GetQueueAttributes
               - sqs:ReceiveMessage
             Resource: !GetAtt ExtractQueue.Arn
+          - Effect: Allow
+            Action:
+              - "sns:*"
+            Resource: "*"
+          - Effect: Allow
+            Action:
+              - "textract:*"
+            Resource: "*"
```

Comment:
Bot actions and resources can probably be locked down more. But it's OK to leave like this to get it running, but don't forget to tighten afterwards.